src/WebJobs.Script.Grpc/Capabilities/Capabilities.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Google.Protobuf.Collections;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc.Capabilities
+{
+    public class Capabilities
+    {
+        private static IDictionary<string, string> _capabilities = new Dictionary<string, string>();
+
+        public static bool IsCapabilityEnabled(string capability)
+        {
+            string isEnabled = "false";
```

Comment:
Thanks for the PR. Thinking more about how to use capabilities,  IsCapabilityEnabled is not needed. Helper you added `GetCapabilityState` is the only one we need.  Delete `IsCapabilityEnabled`